elkserver/install-elkserver.sh

```diff
@@ -452,18 +459,18 @@ if [ $ERROR -ne 0 ]; then
     echoerror "Could not set the default index for Kibana (Error Code: $ERROR)."
 fi
 
-echo "Installing GeoIP index template adjustment"
-curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/redirtraffic- -d@./templates/elasticsearch-template-geoip-es7x.json >> $LOGFILE 2>&1
+echo "Installing Elasticsearch ILM policy"
+curl -X PUT "http://localhost:9200/_ilm/policy/redelk" -H "Content-Type: application/json" -d @./templates/redelk_elasticsearch_ilm.ndjson; done >> $LOGFILE 2>&1
```

Comment:
Currently keeps 365 days of data in the index or max 100Go (should be enough for any red team)

Reply:
Should be fine